types/wordpress__editor__build-types__store__selectors/index.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module '@wordpress/editor/build-types/store/selectors' {
+	export interface WPBlockSelection {
```

Comment:
this doesn't seem necessary, the type exists

Reply:
Confirmed that removing this causes types to fail. Same goes for the other 2 below.